releases/1.6/edge/kubeflow-lite/bundle.yaml

```
@@ -0,0 +1,164 @@
+# Modified from https://github.com/canonical/kubeflow-ci/blob/main/releases/1.4/kubeflow-bundle.yaml
+
+bundle: kubernetes
+name: kubeflow-lite
+applications:
+  admission-webhook:
+    charm: admission-webhook
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: admission-webhook-operator
+  argo-controller:
+    charm: argo-controller
+    channel: 3.3/edge
+    scale: 1
+    _github_repo_name: argo-operators
+  dex-auth:
+    charm: dex-auth
+    channel: 2.31/edge
+    scale: 1
+    trust: true
+    _github_repo_name: dex-auth-operator
+  envoy:
+    charm: envoy
+    channel: 1.20/edge
+    scale: 1
+    _github_repo_name: envoy-operator
+  istio-ingressgateway:
+    charm: istio-gateway
+    channel: 1.11/edge
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
```

If we'e going to use latest/edge, we'll also need:
```suggestion
    _github_repo_name: istio-operators
    options:
      kind: ingress
```